frontend/common/stores/role-store.js

```diff
@@ -0,0 +1,46 @@
+const Dispatcher = require('../dispatcher/dispatcher')
+const BaseStore = require('./base/_store')
+const { getRoles } = require('../services/useRole')
+const { getStore } = require('../store')
+
+const controller = {
+  getRoles: (organisationId) => {
+    store.loading()
+    return getRoles(
```

Comment:
I'm confused as to why we need any of this

Reply:
Sorry, I was using the old way, I already replaced it.